airgun/views/new_contentview.py

```diff
@@ -0,0 +1,82 @@
+from selenium.webdriver.common.keys import Keys
+from wait_for import wait_for
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Table
+from widgetastic.widget import Text
+from widgetastic.widget import TextInput
+from widgetastic.widget import View
+from widgetastic_patternfly import BreadCrumb
+from widgetastic_patternfly4.ouia import Button as PF4Button
+from widgetastic_patternfly4.ouia import PatternflyTable
+
+from airgun.views.common import BaseLoggedInView
+from airgun.views.common import SearchableViewMixin
+from airgun.widgets import ConfirmationDialog
+
+
+
+class NewContentViewTableView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[contains(., 'Content Views')]")
+    create_content_view = PF4Button('OUIA-Generated-Button-primary-1')
+    #TODO: add expandable widget to PFTable
+    table = PatternflyTable(component_id='OUIA-Generated-Table-2')
```

Comment:
I tried `table = PatternflyTable(component_id='OUIA-Generated-Table-2')` but that didn't work. The `component_id` of table was different after performing search operation, you will have to look for a better locator. You can try `table = PatternflyTable(locator='.//table[@data-ouia-component-type="PF4/Table"]')` but this one is also not that robust.